api/src/test/java/ai/djl/util/SecurityManagerTest.java

```diff
@@ -0,0 +1,55 @@
+package ai.djl.util;
+
+import ai.djl.util.cuda.CudaUtils;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.FilePermission;
+import java.security.Permission;
+
+public class SecurityManagerTest {
+
+    @Test
+    public void testGetenv() {
+        // Disable access to system environment
+        SecurityManager sm =
```

Comment:
add @AfterTest function to reset SecurityManager after each test

Reply:
Done